### PR TITLE
Shrinking npm package a bit by not including some folders (e.g. the s…

### PR DESCRIPTION
**Pull request by wmathurin**
…hared/doc folder which has the mobile sdk dev guide)

### CODE REVIEW

File: .npmignore [hunk 1]
```diff
@@ -1,10 +1,17 @@
+external/shared/doc/
+external/shared/gen/
+external/shared/test/
+external/shared/tools/
 external/shared/samples/accounteditor/
 external/shared/samples/contactexplorer/
 external/shared/samples/fileexplorer/
 external/shared/samples/simplesync/
+external/shared/samples/simplesyncreact/
```
Comment (bhariharan) [inline .npmignore]: Is ```simplesyncreact``` committed on ```unstable```? I didn't know.

**Comment by bhariharan** [comment]
:+1:

File: .npmignore (hunk 1 above)
Comment (wmathurin) [inline .npmignore]: It is. The big saving is the doc directory (which has the 500 pages PDF), it causes the tgz to go down from 25 to 21 MB. On the iOS side it's only 1.3 MB now.

Comment (bhariharan) [inline .npmignore]: :+1:

**Pull request merged**